roles/sync/tasks/main.yml

```diff
@@ -4,99 +4,70 @@
   include_vars: "{{ item }}"
   loop: "{{ lookup('first_found', os_fallback, errors='ignore', wantlist=True) }}"
 
-- name: Ensure a list of packages installed
-  become: true
-  become_user: root
-  package:
-    name: "{{ utils_sync }}"
-    state: present
+- name: Download Sync service artefact
+  get_url:
+    url: "{{ downloads.sync_zip_url }}"
+    dest: "{{ download_location }}/sync-dist-6.x-{{ sync.version }}.zip"
+    checksum: sha1:{{ lookup('url', downloads.sync_zip_sha1_checksum_url, username=nexus_user, password=nexus_password) }} # pragma: allowlist secret
+    url_username: "{{ nexus_user }}"
+    url_password: "{{ nexus_password }}"
+
+- name: Extract Sync service to {{ sync_home }}
+  unarchive:
+    src: "{{ download_location }}/sync-dist-6.x-{{ sync.version }}.zip"
+    dest: "{{ download_location }}"
+    remote_src: true
```

Comment:
```suggestion
```

Reply:
I'm not sure I get your point.
To me this is needed: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/unarchive_module.html#parameter-remote_src
Or am I missing something?

Reply:
sorry, I was wrong on the meaning of that attribute - I was thinking exactly the opposite (the file has been downloaded right before - it isn't 'remote')